script/ExecTransaction.s.sol

```diff
@@ -46,27 +50,22 @@ contract ExecTransaction is SafeTxDataBuilder {
         );
     }
 
-    function loadSignatures() internal returns (bytes[] memory signatures) {
-        string[] memory cmd = new string[](2);
-        cmd[0] = "cat";
-        cmd[1] = SIGNATURES_FILE;
-
-        bytes memory res = vm.ffi(cmd); // TODO: use vm.readFile
+    function loadSignatures() internal view returns (bytes[] memory signatures) {
+        bytes memory res = bytes(vm.readFile(SIGNATURES_FILE));
```

Comment:
why not use `readLine` btw?

Reply:
bruh didnt know it existed, it's what i need